### ISSUE TITLE
ethers module not found (hardhat)

### ISSUE THREAD

**Issue by novaknole** [nomiclabs/hardhat#1279]
This is my one of the ts files.

https://www.collabshot.com/show/9542dc

As you can see there's a red line underneath.

My `package.json`.

```js
"devDependencies": {
    "@nomiclabs/hardhat-ethers": "^2.0.0",
    "@nomiclabs/hardhat-waffle": "^2.0.0",
    "@typechain/ethers-v5": "^4.0.0",
    "@types/chai": "^4.2.13",
    "@types/mocha": "^8.0.3",
    "@types/node": "^14.11.8",
    "chai": "^4.2.0",
    "ethereum-waffle": "^3.0.0",
    "ethers": "^5.0.0",
    "hardhat": "^2.0.3",
    "hardhat-abi-exporter": "^2.0.6",
    "hardhat-typechain": "^0.3.3",
    "solidity-coverage": "^0.7.12",
    "ts-generator": "0.1.1",
    "ts-node": "^9.0.0",
    "typechain": "^4.0.0",
    "typescript": "^4.1.2"
  },
```

My `hardhat.config.ts`

```js
import '@nomiclabs/hardhat-ethers'
import '@nomiclabs/hardhat-waffle'
import 'hardhat-abi-exporter'
import 'hardhat-typechain'
import 'solidity-coverage'
```

Why does vscode show red line ? when I put the mouse, it says:  

> node_modules/hardhat/internal/lib/hardhat-lib"' has no exported member 'ethers'

**Comment by alcuadrado** [comment]
It's an issue with vscode. Try starting it. If that doesn't fix it, create the tsconfig file that's recommended [here](https://hardhat.org/guides/typescript.html) and restart it again.

**Comment by novaknole** [comment]
I think `tsconfig` fixed it, but it turns out I have the same problem with typechains.

This is how I made `tsconfig.json`

```js
{
    "compilerOptions": {
      "target": "es2018",
      "module": "commonjs",
      "strict": true,
      "esModuleInterop": true,
      "outDir": "dist"
    },
    "include": ["./scripts", "./test", "./typechain"],
    "files": ["./hardhat.config.ts"]
  }
```

**Comment by novaknole** [comment]
@alcuadrado  I've created an issue here - https://github.com/rhlsthrm/hardhat-typechain/issues/15

**Comment by fvictorio** [comment]
Closing this since I don't think there's anything to do on our side.

**Comment by mohamedmansour** [comment]
I tried following the documentation directly and it says `ethers` not found in `hardhat`
![image](https://user-images.githubusercontent.com/68524/119937108-0d2f9880-bf3f-11eb-8d58-8369f3a86195.png)
```
Module '"hardhat"' has no exported member 'ethers'.ts
```

I checked `hardhat-lib.ts` and all its dependencies, there are no `ethers` being exported. To fix it you **must** install tsconfig so it knows the hardhat config. But the documentation for typescript clearly says not to. https://hardhat.org/guides/typescript.html

I recommend just enforcing all TypeScript devs to create that config.
